.github/workflows/publish.yaml

```diff
@@ -0,0 +1,38 @@
+name: Publish Python Distro to Test PyPi
+
+on: push
+
+jobs:
+  build-and-publish:
+
+    name: Build and Publish
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```

Comment:
[q] it looks like we are trying to run this action when we push a tag, there is a native mechanism to do this.

https://github.com/chatton/github-action-templates/blob/master/.github/workflows/publish-to-pypi.yaml#L3

We can run this when a release is published instead if you think that is better.

Reply:
Even better!